src/components/GlobalLearningResourcesPage/AppliedFilters.tsx

```diff
@@ -22,10 +22,10 @@ const AppliedFilters: React.FC<{
     const currentCategory = loaderOptions[categoryId];
     if (Array.isArray(currentCategory)) {
       const updatedCategory = currentCategory.filter((id) => id !== filterId);
-      setLoaderOptions({
-        ...loaderOptions,
+      setLoaderOptions((prevLoaderOptions: FetchQuickstartsOptions) => ({
```

Comment:
You should not need the type for the argument. It should be inferred from the type of the useState